### PR TITLE
⚙️📝 adding rollback script

### PR DESCRIPTION
**Pull request by filmaj**
Here is my attempt at implementing a rollback script for quickly rolling back to a previous known-good version. The script relies on the current continuous deployment method, which tags repo SHAs with a `known-good-<timestamp>` tag after post-`hlx deploy` tests pass and post-`hlx publish` production end-to-end tests pass. Details on the tagging in the [`deploy_commit_and_publish` circleci job](https://github.com/adobe/developer.adobe.com/blob/master/.circleci/config.yml#L122).

FYI @tripodsan @kptdobe @stefan-guggisberg @trieloff @rofe, in case you have comments, or find this helpful to bring back into helix in some form.

### CODE REVIEW

File: rollback.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e # error out on first failure
+BRANCH=$(git symbolic-ref -q HEAD);
+if [ "$BRANCH" = "refs/heads/master" ]; then
+    #git pull --tags origin master
```
Comment (stevengill) [inline rollback.sh]: You aren't doing a `git pull --tags origin master` yet right? You want to though right?

Comment (filmaj) [reply rollback.sh]: o fuck good catch. woopsie!

**Comment by rofe** [comment]
Looks like a handy helper in case a deployment fails. Having a generic rollback capability in Helix in some shape or form would definitely make sense to me.

**Comment by trieloff** [comment]
Indeed: https://github.com/adobe/helix-cli/issues/81

**Pull request closed without merging**